de-lib/src/main/java/org/iplantc/de/analysis/client/views/AnalysisColumnModel.java

```diff
@@ -129,6 +114,10 @@ public String getPath() {
         endDate.setHeader(appearance.endDate());
 
         status.setHeader(appearance.status());
+        status.setMenuDisabled(true);
+        status.setCell(new AnalysisUserSupportCell());
+        status.setSortable(false);
+        status.setHideable(false);
```

Comment:
Does sorting not work for the status column anymore? If sorting still works on the status value, I don't think we should remove it (the name and date columns use custom cells and they are still sortable).